src/olympia/addons/management/commands/populate_accessibility_category.py

```diff
@@ -0,0 +1,39 @@
+from django.core.management.base import BaseCommand
+
+import olympia.core.logger
+from olympia.addons.models import Addon
+from olympia.constants.base import ADDON_EXTENSION
+from olympia.constants.categories import CATEGORIES
+
+
+log = olympia.core.logger.getLogger('z.addons.populate_accessibility_category')
+
+guids = [
```

Comment:
Just a thought, but I would imagine it would be easier to make this an argument with a default value? What if we need to run it again in prod. Now another PR and another push and another sre ticket...

Reply:
I started going down that route, but ultimately we don't expect to need to do this more than once for a given category we're adding, since it's just to provide some base content, developers can and should tweak categories on their own outside of this specific case. And when we want to add new categories in the future, we'll need to make a change to the constants in the code anyway...